src/read_log.py

```diff
@@ -0,0 +1,24 @@
+"""
+Read Log File
+
+By: Norma Chang
+"""
+import json
+import time
+
+def read_file():
```

Comment:
We don't need this function. Logging should strictly log data. We can read it however we want.